src/panels.cpp

```diff
@@ -1930,12 +1935,13 @@ void panel_manager::draw_adm( const catacurses::window &w, size_t column, size_t
                            panels[i].toggle ?
                            source_index == i && selected ? c_yellow : c_white : c_dark_gray,
                            selected && index - 1 == i ? " %s" : "%s",
-                           selected && index - 1 == i ? saved_name : panels[i].get_name() );
+                           selected && index - 1 == i ? saved_name : _( panels[i].get_name().c_str() ) );
```

Comment:
`saved_name` stays untranslated here and [there](https://github.com/CleverRaven/Cataclysm-DDA/pull/30503/files#diff-a8fe10ec95333cbbc5ab438b7a3b65efR1981).